src/battle_3.c

```diff
@@ -0,0 +1,6431 @@
+extern const u8 gUnknown_082DACC9[];
+extern const u8 gUnknown_082DAC2C[];
+extern const u8 BattleScript_IngrainTurnHeal[];
+extern const u8 BattleScript_LeechSeedTurnDrain[];
+extern const u8 BattleScript_PoisonTurnDmg[];
+extern const u8 BattleScript_BurnTurnDmg[];
+extern const u8 BattleScript_NightmareTurnDmg[];
+extern const u8 BattleScript_CurseTurnDmg[];
+extern const u8 BattleScript_WrapTurnDmg[];
+extern const u8 BattleScript_WrapEnds[];
+extern const u8 gUnknown_082DB234[];
+extern const u8 gUnknown_082DB2A6[];
+extern const u8 BattleScript_ThrashConfuses[];
+extern const u8 BattleScript_DisabledNoMore[];
+extern const u8 BattleScript_EncoredNoMore[];
+extern const u8 BattleScript_YawnMakesAsleep[];
+extern const u8 gUnknown_082DAFE4[];
+extern const u8 gUnknown_082DB8F3[];
+extern const u8 gUnknown_082DAF05[];
+extern const u8 gUnknown_082DAF20[];
+extern const u8 gUnknown_082DA7C4[];
+extern const u8 gUnknown_082DA7CD[];
+extern const u8 BattleScript_MoveUsedWokeUp[];
+extern const u8 BattleScript_MoveUsedIsAsleep[];
+extern const u8 BattleScript_MoveUsedIsFrozen[];
+extern const u8 BattleScript_MoveUsedUnfroze[];
+extern const u8 BattleScript_MoveUsedLoafingAround[];
+extern const u8 BattleScript_MoveUsedMustRecharge[];
+extern const u8 BattleScript_MoveUsedFlinched[];
+extern const u8 BattleScript_MoveUsedIsDisabled[];
+extern const u8 BattleScript_MoveUsedIsTaunted[];
+extern const u8 BattleScript_MoveUsedIsImprisoned[];
+extern const u8 BattleScript_MoveUsedIsConfused[];
+extern const u8 BattleScript_MoveUsedIsConfusedNoMore[];
+extern const u8 BattleScript_MoveUsedIsParalyzed[];
+extern const u8 BattleScript_MoveUsedIsParalyzedCantAttack[];
+extern const u8 BattleScript_MoveUsedIsInLove[];
+extern const u8 BattleScript_BideStoringEnergy[];
+extern const u8 BattleScript_BideAttack[];
+extern const u8 BattleScript_BideNoEnergyToAttack[];
+extern const u8 gUnknown_082DACE7[];
+extern const u8 BattleScript_DrizzleActivates[];
+extern const u8 BattleScript_SandstreamActivates[];
+extern const u8 BattleScript_DroughtActivates[];
+extern const u8 BattleScript_CastformChange[];
+extern const u8 BattleScript_RainDishActivates[];
+extern const u8 BattleScript_ShedSkinActivates[];
+extern const u8 BattleScript_SpeedBoostActivates[];
+extern const u8 BattleScript_SoundproofProtected[];
+extern const u8 BattleScript_MoveHPDrain[];
+extern const u8 BattleScript_MoveHPDrain_PPLoss[];
+extern const u8 BattleScript_FlashFireBoost[];
+extern const u8 BattleScript_FlashFireBoost_PPLoss[];
+extern const u8 BattleScript_ColorChangeActivates[];
+extern const u8 BattleScript_RoughSkinActivates[];
+extern const u8 BattleScript_ApplySecondaryEffect[];
+extern const u8 BattleScript_CuteCharmActivates[];
+extern const u8 gUnknown_082DB68C[];
+extern const u8 BattleScript_SynchronizeActivates[];
+extern const u8 gUnknown_082DB4B8[];
+extern const u8 gUnknown_082DB4C1[];
+extern const u8 BattleScript_TraceActivates[];
+
```

Comment:
why the extra line?

Reply:
Different set of battlescripts, called by the abilities/items. They should be externed in a sepereate battlescripts.h header anyway

Reply:
put a comment elaborating this then.